.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Normally on these issue templates the instructions go into comments. I think that provides 2 benefits:
 1. It's easy for the user to see what should be taken out of the final report and what should be left in (i.e. headings, etc.)
 1. We never see the cruft in the rendered issue, even if the user forgets/doesn't delete the instructions.

For example, notice ESLint's issue templates:
https://github.com/eslint/eslint/blob/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md

Reply:
That makes sense to me too